### DISCUSSION TITLE
Why is GitHub saying I starred a repo in my For You tab even though I never starred the repo?

### DISCUSSION THREAD

**Discussion by Nik-Sharp** [community/community#18933] [Other Feature Feedback, Questions, & Ideas]
Despite the fact that I only have one repository starred, my For You page is saying I starred some random repositories that I've never even heard of.

Example:
![image](https://user-images.githubusercontent.com/96154725/174497841-a0afe055-1c5a-476a-b7e7-81a14d93cda4.png)

**Comment by NisooJadhav** [comment]
**Maybe it is suggesting to you some random repositories(that you might be interested in) based on your starred activities.**

**Reply by marlonrichert** [reply]
But it's suggesting the exact same repo that it says you've starred. So, even if you did star that repo (which you didn't), then that's not at all a helpful suggestion.

**Reply by NisooJadhav** [reply]
Then you have selected starred from the filters, if you select Recommendations it will show, recommended for you

![Screenshot_20220620-185639_Chrome](https://user-images.githubusercontent.com/68807845/174612797-f36a7e24-dccf-4547-a6e8-7687dad3f382.png)

**Reply by marlonrichert** [reply]
Yes, recommendation is correct; you’ll see the same ones as when you go to Explore. However, in the Feed, the text, which explains why it’s recommended for you, is completely incorrect. That is the problem we’re discussing here.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
